SU-SWS/stanford_basic
# D8CORE-941: drop cap styles

# READY FOR REVIEW 

# Summary
WYSIWYG tweaks
_See ticket:  https://stanfordits.atlassian.net/browse/D8CORE-941_

# Needed Review By (Date)
- Sprint end


# Urgency
- Fixes broken stuff
- Not Urgent

# Steps to Test

- Checkout this branch
- Within the WYSIWYG, create a paragraph with a drop-cap style
- save 
- verify it's baseline is on the same baseline as the second line of text in the paragraph, and it's about as tall as those first two lines:

<img width="1341" alt="Screen Shot 2020-01-08 at 12 00 47 PM" src="https://user-images.githubusercontent.com/284440/72011384-94775180-320e-11ea-89bf-116c39d31af9.png">

# Affected Projects or Products
- D8Core (https://github.com/SU-SWS/acsf-cardinalsites)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/D8CORE-941

## Related PRs

## Accessibility Testing

## More Information
By reducing the line-height and increasing the margin-top, I got Firefox and Chrome to look the same.

## Folks to notify
`@sherakama` 
`@kerri-augenstein`

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
